packages/splitter/splitter.stories.tsx

```diff
@@ -193,30 +200,139 @@ Hook.parameters = {
   }
 };
 
-export const Controlled = ({
+export const ManyHooks = ({
   type,
   min,
   max,
   orientation,
   ariaLabel,
-  primaryPaneId,
+  defaultValueNow,
   keyboardStep,
   rtl
```

Comment:
is the `rtl` control meant to remain on this story?

Reply:
Given offline meeting, we will reintroduce `rtl` prop and add `position` prop with `before | after` properties back into the Splitter props.